$MirOS: ports/print/cups/patches/patch-test_run-stp-tests_sh,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- test/run-stp-tests.sh.orig	Fri May  5 16:33:34 2006
+++ test/run-stp-tests.sh	Tue May  9 17:39:40 2006
@@ -30,7 +30,7 @@ argcount=$#
 # Make the IPP test program...
 #
 
-make
+${MAKE_PROGRAM}
 
 #
 # Figure out the proper echo options...
